hack/csv-generate.sh

```diff
@@ -1,26 +1,17 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
 set -e
 
+# shellcheck source=common.sh
+source "$(dirname "$0")/common.sh"
+
 GOROOT=$(go env GOROOT)
 export GOROOT
 
-OLD="4.4.0"
-PREV="4.6.0"
-LATEST="4.7.0"
-LATEST_CHANNEL="4.7"
-CSV_SKIP_RANGE=">=${PREV} <${LATEST}"
+CSV_SKIP_RANGE=">=${PREV_CSV_VERSION} <${CSV_VERSION}"
```

Comment:
Why not move all constants to common?

Reply:
Currently, I moved only constants that used under different scripts, so we can move this one as well, once we will see that it needed.

Reply:
ok